Marlin/utility.h

```diff
@@ -34,6 +34,9 @@ void safe_delay(millis_t ms);
   // Convert unsigned int to string with 12 format
   char* itostr2(const uint8_t &x);
 
+  // Convert uint8_t to string with 123 format
+  char* i8tostr3(const uint8_t &x);
```

Comment:
There's no benefit to passing an 8-bit value as a reference. In terms of saving on parameter passing, it's really only helpful for `float` or `long`, since you save 2 bytes on the stack on AVR.